action/action.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	AntiqueTxType    = 0 // when we first enable web3 tx format, version = 0 is used, all such tx are legacy tx
+	AntiqueTxType    = 0
```

Comment:
since the txType is totally new field, do we still need AntiqueTx? or we can start with `LegacyTxType = 0`?

Reply:
see 2nd commit